packages/common/src/index.ts

```diff
@@ -775,12 +775,12 @@ export type UpdateProject = Omit<
 
 export const getResultValueArray = (
     rows: ResultRow[],
-    onlyRaw: boolean = false,
+    preferRaw: boolean = false,
 ): Record<string, unknown>[] =>
     rows.map((row) =>
         Object.keys(row).reduce<Record<string, unknown>>((acc, key) => {
-            const value = onlyRaw
-                ? row[key]?.value.raw
+            const value = preferRaw
+                ? row[key]?.value.raw ?? row[key]?.value.formatted
```

Comment:
I thought linter was not happy about this nested conditions

Reply:
Same, I have changed this so either are assigned to a variable and easier to read 👌